Packs/QRadar/Integrations/QRadar_v2/QRadar_v2.py

```diff
@@ -529,6 +530,26 @@ def upload_indicators_list_request(self, reference_name, indicators_list):
             "POST", url, params=params, data=json.dumps(indicators_list)
         )
 
+    def get_custom_fields(
+            self, limit: Optional[int] = None) -> List[dict]:
+        url = urljoin(self._server, "api/config/event_sources/custom_properties/regex_properties")
+        headers = self._auth_headers
+        fields = []
+        # Paging
+        start = 0
+        stop = 49
```

Comment:
why is that 49?

Reply:
performance-wise. the API does not limit it but may take a long time to bring back lots of attributes.